server/views/applications/pages/health-needs/learning-difficulties.njk

```diff
@@ -3,8 +3,183 @@
 
 {% block questions %}
 
-  <p class="govuk-body">
-    Learning difficulties page
+  {% set learningNeedsFollowUp %}
+  {{
+      formPageInput(
```

Comment:
I believe all the `formPageInput`s should be `formPageTextArea`s for this template?